.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.11.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+#   webhint:
+#     name: Webhint
+#     runs-on: ubuntu-22.04
+#     steps:
+#       - uses: actions/checkout@v3
+#       - uses: actions/setup-node@v3
+#         with:
+#           node-version: "18.x"
+#       - name: Setup Webhint
+#         run: |
+#           npm install --save-dev hint@7.x
+#           [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+#       - name: Webhint Report
+#         run: npx hint .
```

Comment:
- [ ] Please don't make any changes in your `.github/workflows` **linters.yml** file because it will check your code quality and make you aware regarding your code is standardized or not before merging your code into the *main* branch. As I can see that you have commented out the `webhint` linter from your file and because of that your linter test pass ✅ not appear on the PR, there should be 4 checks instead of 3.

### Requirement
![image](https://github.com/nouman17610/hello-microverse/assets/66476329/e57b1fb8-89e1-423b-81d5-461432d9231b)

Reply:
@skyv26, thank you for your wonderful review. The reason why I commented "web hint" linters.yml was that, the settings are not working properly and we were told remove it, so for me to remove I commented it but now I remove it.
Once again, thanks 👍 .

Reply:
@yayaismaail you are welcome but `webhint` problem is resolved now make sure that next time you don't miss it.